ndmg/utils/gen_utils.py

```diff
@@ -42,6 +41,20 @@
 from scipy.sparse import lil_matrix
 
 
+def show_template_bundles(final_streamlines, template_path, fname):
+    import nibabel as nib
+    from fury import actor, window
+    renderer = window.Renderer()
+    template_img_data = nib.load(template_path).get_data().astype('bool')
+    template_actor = actor.contour_from_roi(template_img_data,
+                                            color=(50, 50, 50), opacity=0.05)
+    renderer.add(template_actor)
+    lines_actor = actor.streamtube(final_streamlines, window.colors.orange,
+                                   linewidth=0.3)
+    renderer.add(lines_actor)
+    window.record(renderer, n_frames=1, out_path=fname, size=(900, 900))
+    return
+
```

Comment:
pep-8 doesn't generally like imports inside of functions: https://www.python.org/dev/peps/pep-0008/#imports
doesn't matter too much in this case, but just FYI

Reply:
For sure, I'm used to this from working with nipype where function nodes have their own name spaces!